src/DotLiquid/Context.cs

```diff
@@ -91,6 +93,39 @@ public Context
             SquashInstanceAssignsWithEnvironments();

```

Comment:
Avoid code duplication by having one of the two methods call the other one instead.